packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -102,8 +102,9 @@ const AccountSwitcher = observer(({ history, is_mobile, is_visible }) => {
         if (is_positions_drawer_on) {
             togglePositionsDrawer(); // TODO: hide drawer inside logout, once it is a mobx action
         }
+        await logoutClient();
+        window.location.href = getStaticUrl('/');
         history.push(routes.index);
```

Comment:
@rupato-deriv these 2 lines are conflicting. We need to keep only one location/path switcher in here.